libs/remix-core-plugin/src/lib/compiler-artefacts.ts

```diff
@@ -72,51 +72,61 @@ export class CompilerArtefacts extends Plugin {
     return contractsData
   }
 
-  async getArtefactsFromFE (path, contractName) {
+  getAllContractArtefactsfromOutput (contractsOutput, contractName) {
+    const contractArtefacts = {}
+    for (const filename in contractsOutput) {
+      if(Object.keys(contractsOutput[filename]).includes(contractName)) contractArtefacts[filename + ':' + contractName] = contractsOutput[filename][contractName]
+    }
+    return contractArtefacts
+  }
+
+  async getArtefactsFromFE (path, contractName, contractArtefacts) {
```

Comment:
could you set it as private if it's meant to be used only in that class?

Reply:
same above

Reply:
done